src/Language/PureScript/AST/Exported.hs

```diff
@@ -144,3 +141,16 @@ isDctorExported ident (Just exps) ctor = test `any` exps
   test (TypeRef _ ident' Nothing) = ident == ident'
   test (TypeRef _ ident' (Just ctors)) = ident == ident' && ctor `elem` ctors
   test _ = False
+
```

Comment:
Oh, I just saw your comment about this code being `docs & repl` specific anyway. In that case I'm not opposed to doing it this way, but I still think having an actual document structure derived from the export section would be a nice way to go about extracting documentation.